packages/react-instantsearch-core-next/src/useIndex.ts

```diff
@@ -1,4 +1,4 @@
-import index from 'instantsearch.js/cjs/widgets/index/index';
+import index from 'instantsearch.js/es/widgets/index/index';
```

Comment:
we should add a local eslint rule to only use es imports so this can't happen by accident (although bundle size would show it I guess)

Reply:
Good idea! See b2e23190855afa617b3e09f56e2330e1c17e11c7.